### ISSUE TITLE
Coco Validation Set Needed

### ISSUE THREAD

**Issue by kurtisdavid** [google-research-datasets/bim#4]
Hello,

Currently I'm trying to setup the dataset. I've already ran the **downloads_datasets.sh** and currently running **construct_bim_dataset.py**. However, I run into this message:

`Could not find directory /home/kurtis/bim/data/coco/images/val`. 

Do I need to also download val2017 for coco? Because the downloads_datasets.sh script only downloads the training set

**Comment by sherryy** [comment]
Fixed. This issue should have not affected the images being constructed for BIM though. Only the images for running TCAV were affected.
